Assets/ModScripts/Submodules/LogicalColorCombinations.cs

```diff
@@ -0,0 +1,316 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.NetworkInformation;
+using System.Security.Principal;
+using UnityEngine;
+using KModkit;
+using Random = UnityEngine.Random;
```

Comment:
```suggestion
﻿using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using UnityEngine;
using Random = UnityEngine.Random;
```
Some of these usings aren't necessary